### PR TITLE
手工麻糍

### PR DESCRIPTION
**Pull request by chrisnico**

### CODE REVIEW

**Comment by immoonancient** [comment]
#193

**Comment by immoonancient** [comment]
@kdliu86 : 稿件貌似有比较多的缺漏，审核的时候请注意对照原视频

File: subtitles/xue-yu/20190915-shou-gong-ma-ci.sbv [hunk 1]
```diff
@@ -0,0 +1,361 @@
+# https://www.youtube.com/watch?v=syAahnaiwNE
+
+
+# 福建·泉州
+
```
[the remark this answers is not in the document]
Comment (immoonancient) [reply subtitles/xue-yu/20190915-shou-gong-ma-ci.sbv]:
```suggestion
(Quanzhou, Fujian)
```

File: subtitles/xue-yu/20190915-shou-gong-ma-ci.sbv [hunk 2]
```diff
@@ -0,0 +1,361 @@
+# https://www.youtube.com/watch?v=syAahnaiwNE
+
+
```
Comment (immoonancient) [inline subtitles/xue-yu/20190915-shou-gong-ma-ci.sbv]:
```suggestion
00:00:01.880,00:00:04.920
```

File: subtitles/xue-yu/20190915-shou-gong-ma-ci.sbv [hunk 3]
```diff
@@ -0,0 +1,361 @@
...
+# 堂食请自取
+Please pick up yourself for eat-in
+
+0:00:41.540,0:00:43.260
+# 额 糟糕
+oh blast
+
+0:00:43.260,0:00:45.800
+# 每当看菜单的时候 我都会变的很纠结
+I struggle every time when I read the menu
```
[the remark this answers is not in the document]
Comment (haoyuanli) [reply subtitles/xue-yu/20190915-shou-gong-ma-ci.sbv]: Great advice, how about 'Whenever I see the menu, I get anxious'

[the remark this answers is not in the document]
Comment (immoonancient) [reply subtitles/xue-yu/20190915-shou-gong-ma-ci.sbv]: > Great advice, how about 'Whenever I see the menu, I get anxious'

SGTM

**Pull request merged**